<p>
    The DML statement, can not be inside a for loop
</p>

<h2>Noncompliant Code Example</h2>

<pre>
for(int i = 0; i < 10; i++) {
    insert newAcct; 
}
</pre>

<h2>Compliant Solution</h2>

<pre>
for(int i = 0; i < 10; i++) {
    ...
}
insert newAcct; 
</pre>
