<!--
  ~ Copyright (c) Fundacion Jala. All rights reserved.
  ~ Licensed under the MIT license. See LICENSE file in the project root for full license information.
  -->

<p>
    It's bad practice to use incorrectly assert methods.
</p>

<h2>Noncompliant Code Example</h2>

<pre>
@isTest public void testContent(){
    ...
    System.assertEquals(name, name);
    ...
    System.assert(true);
}
</pre>

<h2>Compliant Solution</h2>

<pre>
@isTest public void testContent(){
    ...
    System.assertEquals(name, 'Jalasoft');
    ...
    System.assert(book.isOpen());
}
</pre>
