<!--
  ~ Copyright (c) Fundacion Jala. All rights reserved.
  ~ Licensed under the MIT license. See LICENSE file in the project root for full license information.
  -->

<p>
Defining naming conventions is key to make it possible for a team to efficiently collaborate. 
All class names must match the provided regular expression.
</p>

<h2>Noncompliant Code Example</h2>
<p>With default provided regular expression: ^[A-Z_][a-zA-Z0-9]+$</p>

<pre>
class myClass:
	...
</pre>

<h2>Compliant Solution</h2>

<pre>
class MyClass:
	...
</pre>
