<!--
  ~ Copyright (c) Fundacion Jala. All rights reserved.
  ~ Licensed under the MIT license. See LICENSE file in the project root for full license information.
  -->

<p>
    The method is deprecated and your body is not empty, it can lead to problems throughout.
</p>

<h2>Noncompliant Code Example</h2>

<pre>
@deprecated
public void init(){
    int min = 0;
}
</pre>

<h2>Compliant Solution</h2>

<pre>
@deprecated
public void init(){    
}
</pre>
