<p>
Sharing some naming conventions is a key point to make it possible for a team to efficiently collaborate. 
This rule allows to check that all class names match a provided regular expression.
</p>

<h2>Noncompliant Code Example</h2>
<p>With default provided regular expression: ^[A-Z_][a-zA-Z0-9]+$</p>

<pre>
class myClass:
	...
</pre>

<h2>Compliant Solution</h2>

<pre>
class MyClass:
	...
</pre>
