<!--
  ~ Copyright (c) Fundacion Jala. All rights reserved.
  ~ Licensed under the MIT license. See LICENSE file in the project root for full license information.
  -->

<p>
    A DML statement cannot be inside a constructor.
</p>

<h2>Noncompliant Code Example</h2>

<pre>
public MyClass(){
    insert newAcct; 
}
</pre>

<h2>Compliant Solution</h2>

<pre>
public MyClass(){
    ...
}
public void insertAcct(){
    insert newAcct;
}
</pre>
