<!--
  ~ Copyright (c) Fundacion Jala. All rights reserved.
  ~ Licensed under the MIT license. See LICENSE file in the project root for full license information.
  -->

<p>
    It is bad practice to hard-code IDs in methods or constructors.
</p>

<h2>Noncompliant Code Example</h2>

<pre>
public void method (ID newRecordID) {
    for (Account current : listOfcounts){
        if (current.getId == '34345446546546a') {

        }
    }
    method('3266sd35435sd6a');
}
</pre>

<h2>Compliant Solution</h2>

<pre>
public void method (ID newRecordID) {
    for (Account current : listOfcounts){
        if (current.getId == newRecordID) {

        }
    }
    
    otherMethod(newRecordID);
}
</pre>
