<!--
  ~ Copyright (c) Fundacion Jala. All rights reserved.
  ~ Licensed under the MIT license. See LICENSE file in the project root for full license information.
  -->

<p>
    You must write at least one assertion and your method should have 
    testMethod keyword.
</p>

<h2>Noncompliant Code Example</h2>

<pre>
@isTest
public class SampleTest() {
    static testMethod void someTestMethod() {
    //a block of code, without System.assert 
    }
}

</pre>

<pre>
@isTest
public class SampleTest() {
    static void someTestMethod() {
    //block of code.
    System.assert(something); // this is a test context.
    }
}

</pre>

<h2>Compliant Solution</h2>

<pre>
@isTest
public class SampleTest() {
    static testMethod void someTestMethod() {
    ...
    System.assert(something);
    }

    ...

    static testMethod void someTestMethod() {
    ...
    System.assertEquals(something, otherThing, 'message');
    }
}
</pre>

<pre>
public class SampleTest() {
    static testMethod void someTestMethod() {
    ...
    System.assert(something);
    }

    ...

    static testMethod void someTestMethod() {
    ...
    System.assertEquals(something, otherThing, 'message');
    }
}
</pre>


