<!--
  ~ Copyright (c) Fundacion Jala. All rights reserved.
  ~ Licensed under the MIT license. See LICENSE file in the project root for full license information.
  -->

<p>
TestMethods should be declared only in test classes to maintain organized code.
</p>

<h2>Noncompliant Code Example</h2>
<pre>
public with sharing SomeClass {
...
public testMethod void someTestMethod() {...} // this isn't a test class, so it shouldn't have a test method
}
</pre>