<!--
  ~ Copyright (c) Fundacion Jala. All rights reserved.
  ~ Licensed under the MIT license. See LICENSE file in the project root for full license information.
  -->

<p>
Setting "@isTest(seeAllData = true)" must be used carefully, or not at all, since it can represent a security breach by exposing data.
</p>

<h2>Noncompliant Code Example</h2>
<pre>
@isTest(seeAllData = true)
public with sharing SomeTestClass {
...
}
</pre>

<h2>Compliant solution</h2>
<pre>
@isTest
public with sharing SomeTestClass {
...
}
</pre>