<!--
  ~ Copyright (c) Fundacion Jala. All rights reserved.
  ~ Licensed under the MIT license. See LICENSE file in the project root for full license information.
  -->

  <p>For better readability avoid too many parameters in methods/constructors.</p>
  
  <h2>Noncompliant Code Example</h2>
  <p>The maximum number of parameters in method/constructor is 5.</p>
  
  <pre>
  void someMethod(integer arg1, integer arg2, integer arg3, integer arg4, integer arg5, integer arg6) { // violation
      }
  
  class SampleClass {
      SampleClass(integer arg1, integer arg2, integer arg3, integer arg4, integer arg5, integer arg6, integer arg7) { // violation
      }
  }
  </pre>
  
  <h2>Compliant Solution</h2>
  
  <pre>
        void someMethod(integer arg1) {
        }

    class SampleClass {
        SampleClass(integer arg1) {
        }
    }
  </pre>
  