<!--
  ~ Copyright (c) Fundacion Jala. All rights reserved.
  ~ Licensed under the MIT license. See LICENSE file in the project root for full license information.
  -->

<p>
SOQL statements should always have a defined LIMIT because query results can be larger than expected and most records will not be used.
</p>

<h2>Noncompliant Code Example</h2>
<pre>
public void someMethod {
...
Accounts[] accs = [SELECT dato FROM table1];
}
</pre>

<h2>Compliant Solution</h2>
<pre>
public void someMethod {
...
Accounts[] accs = [SELECT dato FROM table1 LIMIT 50];
}
</pre>