<!--
  ~ Copyright (c) Fundacion Jala. All rights reserved.
  ~ Licensed under the MIT license. See LICENSE file in the project root for full license information.
  -->

<p>
SOQL statements should not be in loops.
</p>

<h2>Noncompliant Code Example</h2>
<pre>
public void someMethod {
	...
	while (true) {
		Contact c = new Contact(Account = [SELECT Name FROM Account WHERE NumberOfEmployees > 10 LIMIT 1]);
	}
}
</pre>

<h2>Compliant Solution</h2>
<pre>
public void someMethod {
	...
	Contact c = new Contact(Account = [SELECT Name FROM Account WHERE NumberOfEmployees > 10 LIMIT 1]);
}
</pre>